tests/test_map.py

```diff
@@ -46,7 +47,7 @@ def test_map_to_seq(xs: Dict[str, int]):
     items: ItemsView[str, int] = xs.items()
     ys = map.of_seq(items)
     zs = pipe(ys, map.to_seq)
-    assert list(xs) == list(zs)
```

Comment:
this was not silently failing due to the duplicate test below

Reply:
Fixed